eos-update-flatpak-repos

```diff
@@ -152,7 +155,7 @@ def _remove_runtimes():
         logging.info("Removing runtime {} from {}"
                      .format(refspec, inst.get_path().get_path()))
         subprocess.check_call(['flatpak', 'uninstall', '--system',
-                               '--runtime', refspec])
+                               '--runtime', '--noninteractive', refspec])
```

Comment:
Unsure if it should use `-y` instead or even if really needed (but it seems to be from my tests, otherwise the systemd service fails when getting a prompt).